src/PurgeClient/VarnishPurgeClient.php

```diff
@@ -44,20 +45,20 @@ public function purge($tags)
             return;
         }
 
-        // As key only support one tag being invalidated at a time, we loop.
-        // These will be queued by FOS\HttpCache\ProxyClient\Varnish and handled on kernel.terminate.
-        foreach (array_unique((array)$tags) as $tag) {
-            if (is_numeric($tag)) {
-                $tag = 'location-' . $tag;
-            }
+        // For 1.x BC make sure to map any int to location id tag
+        $tags = array_unique(array_map(static function ($tag) {
+            return is_numeric($tag) ? 'location-' . $tag : $tag;
+        },
+            (array)$tags
+        ));
 
-            $headers = [
-                'key' => $tag,
-                'Host' => empty($_SERVER['SERVER_NAME']) ? parse_url($this->configResolver->getParameter('http_cache.purge_servers')[0], PHP_URL_HOST) : $_SERVER['SERVER_NAME'],
-            ];
-
-            $headers = $this->addPurgeAuthHeader($headers);
+        $headers = $this->getPurgeHeaders();
+        $chunkSize = $this->determineTagsPerHeader($tags);
 
+        // NB! Thish requries varnish-moduls 0.10.2 or higher, 0.9.x only supported purging one tag at a time
```

Comment:
Have you confirmed that Platform.sh supports 0.10.2?

Reply:
No, but afaik varnish-moduels 0.9 won't compile on Varnish 5: https://github.com/varnish/varnish-modules/issues/44